src/types/account/index.ts

```diff
@@ -96,6 +98,12 @@ export type AccountSettingsAttrs = {
 	[key: string]: string | number | Array<string | number> | undefined;
 };
 
+export type UpdateSettings = (settingsMods: Partial<Mods>) => void;
+export type UpdateAccount = (
+	accountMods: Partial<Mods>,
+	response: Pick<EditSettingsBatchResponse, 'CreateIdentityResponse'>
```

Comment:
This arg is misleading in my opinion. It is called response, but it could be better to name it identities, and make it unrelated to the network response

Reply:
should be better now